hbase-it/src/test/java/org/apache/hadoop/hbase/DistributedHBaseCluster.java

```diff
@@ -33,17 +33,23 @@
 import org.apache.hadoop.hbase.client.ConnectionFactory;
 import org.apache.hadoop.hbase.client.RegionInfo;
 import org.apache.hadoop.hbase.client.RegionLocator;
+import org.apache.hadoop.hbase.testing.HBaseCluster;
 import org.apache.hadoop.hbase.util.Bytes;
 import org.apache.hadoop.hbase.util.EnvironmentEdgeManager;
 import org.apache.hadoop.hbase.util.Threads;
 import org.apache.yetus.audience.InterfaceAudience;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * Manages the interactions with an already deployed distributed cluster (as opposed to
- * a pseudo-distributed, or mini/local cluster). This is used by integration and system tests.
+ * Manages the interactions with an already deployed distributed cluster (as opposed to a
+ * pseudo-distributed, or mini/local cluster). This is used by integration and system tests.
  */
 @InterfaceAudience.Private
 public class DistributedHBaseCluster extends HBaseCluster {
+
+  private static final Logger LOG = LoggerFactory.getLogger(DistributedHBaseCluster.class);
```

Comment:
Used?

Reply:
It used to use the one in HBaseCluster, which I do not think is a good practise...